### PR TITLE
Remove margin of body

### PR DESCRIPTION
**Pull request by JeromeDeLeon**
To make it look like this

Screenshot from 2019-12-26 11-35-55

Instead of this
Screenshot from 2019-12-26 11-38-28

### CODE REVIEW

**Comment by penge** [comment]
Cool, haven't realized this is how it looks on Windows.
(I have no access to other than Chromebook, sorry about that, but I will gladly merge this!)

Could you try two things for me? (without comments)

### 1

```css
/* Same as you set */

body {
  margin: 0;
} 


/* Additionally */

#textarea {
  padding: 13px;

  /*
    padding is originally set to 5px here but
    because we set body margin to 0px
    (which I suppose is by default 8px)
    I have set here 5 + 8 = 13px to get
    the same look as before while scrollbar
    is to the right.
  */
}

#settings {
  right: 0;  /* before, it was 5px; */
  padding: 20px 25px 20px 20px;  /* before, it was 20px; */
}
```

I guess this should fix the scrollbar position while maintaining the same look.

### 2

How it looks in the Options page? Is the scrollbar ok there?

You can try update `options.css` to be:

```css
body {
  margin: 0;
}
```

To see if it does the change or make it look better.

Thank you.

**Comment by JeromeDeLeon** [comment]
For the Options page, I don't think it's necessary because the scrollbar is on `body` tag, but for consistency, we can make it the same. I have tried it and both seems fine.

**Comment by JeromeDeLeon** [comment]
For font size options, I think it's better if you move it out in the Options page as well.

File: notes.css [hunk 1]
```diff
@@ -30,8 +31,9 @@ body {
   font-family: monospace;
   font-weight: bold;
   bottom: 0;
+  right: 0;
   right: 5px;
```
Comment (penge) [inline notes.css]: I think this line should be removed.

**Comment by penge** [comment]
I like it so far. I have made one comment to the changes.

Regarding font size, it is something I consider whether to keep or drop completely.

If the font size would be in the Options page only:
- pro: more space for notes
- con: need to go to the Options page every time to change it

If the font size would be dropped completely:
- pro: advanced users are good with `Ctrl+` and `Ctrl-`
- con: basic users might not know about the shortcuts
- con: font size is not saved for later visits

**And so I am thinking about the following compromise, which would be:**
Adding to the Options page whether to show or hide the slider.
Some might keep the slider visible (making changes often, could be a presentation for example).
Some might hide the slider once it is set.
Options and Notes can be open at the same time to tweak the font size one might want to keep.
Notes itself would be used to preview the changes.

What do you think about that?

**Comment by JeromeDeLeon** [comment]
You already hide the theming so why not the font size? I mean, I was shocked when I opened it after vacation and found that theming was nowhere to be found until I opened Options page, so same goes for font size slider. Just tell users about what you changed, or the features of this extensions and we're good to go. (I mean all of this in a good way btw :smile:)

**Comment by JeromeDeLeon** [comment]
I have something in mind. I don't know if it's good for you. Anyway, what I have is that when document loaded, you let atleast 3-5s passes before lowering the opacity of settings, and for the rest, only lower the opacity if the mouse hovers on it or something. That way, we can keep writing without minding about the overlapping because the text is still readable.

**Comment by JeromeDeLeon** [comment]
That is, if you don't want to move out the font size slider in the Options page.

**Comment by penge** [comment]
Yeah 😁

I am thinking to add something like welcome page that would on the first visit when the new version is installed, show Hey dude this is new this is new this is 2.1 this is what changed from 2.0 if you scroll down you can see what changed since 1.0.

Basically very concise changelog on the first visit of a new update.

**Comment by penge** [comment]
3-5 sec sounds good.

But yo convinced me. Moving everything to Options make it cleaner 🤠

I shall do it later today also merge this PR. Thanks for suggestions.

**Comment by JeromeDeLeon** [comment]
You're welcome! :+1:
This extension is great. Since moving to Ubuntu, I missed notepad by just pressing the shortcut. Easy notes. Fast. Simple yet Elegant. :smiley:

**Pull request merged**